README.md

```diff
@@ -1,9 +1,33 @@
-<h1>Trompa Contributor Environment Python Client</h1>
+# Trompa Contributor Environment Python Client
 
 <!-- <h2>Pritish Chandna, Merlijn Blaauw, Jordi Bonada, Emilia Gómez</h2> -->
 
-<h2>Music Technology Group, Universitat Pompeu Fabra, Barcelona</h2>
+## Music Technology Group, Universitat Pompeu Fabra, Barcelona
 
 A python library to read from and write to the Trompa CE
-<h3>Installation</h3>
-To install, run <pre><code>setup.py</code></pre> to install the packages required.
\ No newline at end of file
+### Installation 
+To install, run setup.py to install the packages required.
+
+
+<!-- <h2>Application Side Functionality</h2>
```

Comment:
what are these comment markers for?

Reply:
I wanted to put some instructions on how to use, similar to the other repository, but I'm not really sure what to put. Could just remove it for now.